aead/rustfmt.toml

```diff
@@ -1 +0,0 @@
-merge_imports = true
```

Comment:
This is super annoying for stable Rust users:

```
$ cargo fmt
Warning: can't set `merge_imports = true`, unstable features are only available in nightly channel.
```

Reply:
I think you can remove this file and formatting can be changed later.